bom/pom.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
 
   <properties>
-    <slf4jVersion>1.7.31</slf4jVersion>
+    <slf4jVersion>1.7.32</slf4jVersion>
```

Comment:
Best handled in a separate PR linking to the relevant changelogs (Dependabot will probably open one once we clear the backlog.)

Reply:
> Dependabot will probably open one once we clear the backlog.

As promised: #5631